benchmarking/blackbox_repository/blackbox_surrogate.py

```diff
@@ -31,6 +31,7 @@ def __init__(
             fidelity_space: Optional[Dict] = None,
             fidelity_values: Optional[np.array] = None,
             surrogate=KNeighborsRegressor(n_neighbors=1),
```

Comment:
I'd prefer =None here, and then set the default inside. It is a better pattern.

Reply:
Updated.